packages/Webkul/Shop/src/Mail/Customer/RegistrationNotification.php

```diff
@@ -14,18 +15,53 @@ class RegistrationNotification extends Mailable
      * Create a new mailable instance.
      *
      * @param  \Webkul\Customer\Contracts\Customer  $customer
+     * @param  string  $notify
      * @return void
      */
-    public function __construct(public $customer)
-    {
+    public function __construct(
+        public $customer,
+        public $notify
+    ) {
     }
 
     /**
-     * Build the message.
+     * Build & send registration notification.
      *
      * @return $this
+     *
+     * @throws \Exception
      */
     public function build()
+    {
+        if ($this->notify === 'admin') {
+            return $this->sendToAdmin();
+        } elseif ($this->notify === 'customer') {
+            return $this->sendToCustomer();
+        }
+
+        // Default action if $notify value doesn't match 'admin' or 'customer'
+        Log::error('Invalid notification target: ' . $this->notify);
+    }
+
+    /**
+     * Notify admin with email.
+     *
+     * @return $this
+     */
+    protected function sendToAdmin()
+    {
+        return $this->from(core()->getSenderEmailDetails()['email'], core()->getSenderEmailDetails()['name'])
+            ->to(core()->getAdminEmailDetails()['email'])
+            ->subject(trans('shop::app.emails.customers.admin.registration.subject'))
+            ->view('shop::emails.customers.notify-admin-about-registration');
+    }
```

Comment:
For the separation of concern, this mail template will call from the admin package and all related translations should be moved to the admin package.

You can take reference of order create mail.